### PR TITLE
Box nades code

### PR DESCRIPTION
**Pull request by Danaleja2005**
Code boxes of grenades (HIDP AND M15)
 
@Ghommie The cost of the crate are of 100 points

### CODE REVIEW

**Comment by Alterist** [comment]
Travis is not giving you a check, if the issue isn't with the code, try opening and closing the pr for it to run again

**Comment by Danaleja2005** [comment]
I do not understand why it gives error in the check, I see everything well in the code

**Comment by Ghommie** [comment]
Probably just travis borked up her.
Anyway, what I were trying to say on the previous PR is that the x5 packs of incendiary/M15 were originally meant to be pricier than the standard, though these box versions all have the same price and amount.

**Comment by Danaleja2005** [comment]
The price of boxes x25 is 100, and that of boxes x5 is 40

**Comment by Ghommie** [comment]
The 100 points for x25 grenades was originally intended for HEDPs, which cost 20 points per 5 ones.

**Comment by Danaleja2005** [comment]
So, how much do you think the cost should be?

**Comment by Ghommie** [comment]
120 per 20, that's 5 free nades. It's mostly up to you and maintainers anyway.

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -462,6 +462,27 @@ WEAPONS
 	containername = "\improper explosive HEDP grenade crate (WARNING)"
 	group = "Weapons"
 
+/datum/supply_packs/explosives_hidp
+ 	name = "M40 HIDP incendiary grenade box crate (x25)"
```
Comment (DominikPanic) [inline code/datums/supplypacks.dm]: The problem may not be obvious at first, but my intuition was right - I used the "convert indentation to tabs" feature of Sublime Text and this is where a space is, at the beginning of the line.
gitkraken_2019-01-11_14-01-22

File: code/game/machinery/vending/marine_vending.dm [hunk 2]
```diff
@@ -111,6 +111,8 @@
 					// /obj/item/weapon/gun/shotgun/combat = 1,
 					/obj/item/explosive/mine = 2,
 					/obj/item/storage/box/nade_box = 2,
+					/obj/item/storage/box/nade_box/M15 = 1 ,
+ 					/obj/item/storage/box/nade_box/incendiary = 0,
```
Comment (DominikPanic) [inline code/game/machinery/vending/marine_vending.dm]: If the default value is to be zero and you want to still keep it useful for admin events, you should move it under the contraband list.

File: code/game/objects/items/storage/boxes.dm [hunk 3]
```diff
@@ -610,9 +610,20 @@
...
+	desc = "A secure box holding 25 M40 HIDP grenades."
+	icon_state = "nade_HIDP_box"
+	can_hold = list("/obj/item/explosive/grenade/incendiary")
+	grenade_type = /obj/item/explosive/grenade/incendiary
 
 
-
+/obj/item/storage/box/nade_box/M15
+	name = "\improper M15 grenade box"
+	desc = "A secure box holding 25 M15 fragmentation grenades."
```
Comment (DominikPanic) [inline code/game/objects/items/storage/boxes.dm]: I'd suggest starting off with just 12 grenades for balancing reasons.

File: code/game/objects/items/storage/boxes.dm [hunk 4]
```diff
@@ -610,9 +610,20 @@
 	can_hold = list("/obj/item/explosive/grenade/chem_grenade/teargas")
 	grenade_type = /obj/item/explosive/grenade/chem_grenade/teargas
 
+ /obj/item/storage/box/nade_box/incendiary
+	name = "\improper M40 HIDP grenade box"
+	desc = "A secure box holding 25 M40 HIDP grenades."
```
Comment (DominikPanic) [inline code/game/objects/items/storage/boxes.dm]: I'd suggest starting off with just 12 grenades for balancing reasons.

**Comment by DominikPanic** [comment]
Also I would suggest leaving either closing the other icon PR or removing the icon changes, whichever you prefer since both PRs do the same thing at the moment.

**Pull request closed without merging**